### PR TITLE
C1 netsuite export settings content fix

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/branding/c1-contents-config.ts [hunk 1]
```diff
@@ -19,13 +19,24 @@ export const c1Contents = {
...
+                contentText: 'Configure how and when expenses from expense management can be exported to Netsuite.',
                 corporateCard: {
+                    creditCardExpenseLabel: 'How should the expenses be exported?',
                     cccExpenseBankAccountSubLabel: 'The selected expense payment type will be added to the corporate credit card expenses exported from ' + brandingConfig.brandName + ' to Netsuite.',
-                    creditCardExportTypeSubLabel: '',
+                    creditCardExportTypeSubLabel: 'Expenses can either be exported as single line items (i.e., expenses) or as a grouped report with multiple line items (i.e., expense reports).',
                     expenseState: '',
-                    creditCardExpenseSubLabel: '',
-                    cccExpenseStateSubLabel: 'You can export expenses either when they\'re awaiting payment after approval (Approved) or when the payment has been settled (Closed).'
+                    creditCardExpenseSubLabel: 'Choose the type of NetSuite transaction to export your expenses.',
```
Comment (ashwin1111) [inline src/app/branding/c1-contents-config.ts:28]: use NetSuite or Netsuite everywhere consistently (S caps or small - please confirm with Shwetabh)

File: src/app/branding/c1-contents-config.ts [hunk 2]
```diff
@@ -19,13 +19,24 @@ export const c1Contents = {
...
+                    cccExpenseStateSubLabel: 'You can export expenses either when they\'re awaiting payment after approval (Approved) or when the payment has been settled (Closed).',
```
Comment (ashwin1111) [inline src/app/branding/c1-contents-config.ts:29]: (Approved), (Closed) -> check qbo/intacct if it's caps

File: src/app/branding/c1-contents-config.ts [hunk 3]
```diff
@@ -19,13 +19,24 @@ export const c1Contents = {
...
+                    creditCardExpenseStateLabel: 'How should expenses be labeled  before exporting from expense management',
+                    creditCardExpenseStateSubLabel: 'You can choose to only export expenses when they\'ve been labeled approved or closed.',
+                    defaultCCCAccountLabel: 'Set Default Credit Card Account as',
```
Comment (ashwin1111) [inline src/app/branding/c1-contents-config.ts:32]: sentence case

File: src/app/branding/c1-contents-config.ts [hunk 4]
```diff
@@ -19,13 +19,24 @@ export const c1Contents = {
...
+                    defaultCCCAccountPlaceholder: 'Select Default Credit Card Account',
```
Comment (ashwin1111) [inline src/app/branding/c1-contents-config.ts:33]: same

File: src/app/branding/c1-contents-config.ts [hunk 5]
```diff
@@ -19,13 +19,24 @@ export const c1Contents = {
...
+                    defaultDebitCardAccountLabel: 'Set Default Debit Card Account as',
```
Comment (ashwin1111) [inline src/app/branding/c1-contents-config.ts:34]: same everywhere

**Comment by DhaaraniCIT** [comment]
PR comments are fixed in [https://github.com/fylein/fyle-integrations-app/pull/760/commits/4ca86aafd2d25a5f99c91815b143538b1aa2e743](url)

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**